woodwork/tests/accessor/test_table_accessor.py

```diff
@@ -2456,7 +2459,11 @@ def test_accessor_schema_properties(sample_df):
         assert prop_from_accessor == prop_from_schema
 
         # Assumes we don't have setters for any of these attributes
-        error = "can't set attribute"
+        if Version(python_version().strip(punctuation)) < Version("3.11.0"):
```

Comment:
We need to strip because the `python_version` call returns `3.11.0+`
I preferred this over `sys.version` which returns a larger, more detailed string that needs to be parsed.